<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProjectDialog</class>
 <widget class="QDialog" name="AddProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un projet</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="leNameProject">
     <property name="toolTip">
      <string>Nom du projet</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbDescription">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbProjectName">
     <property name="text">
      <string>Projet</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="leDescription">
     <property name="toolTip">
      <string>Description du contenu du projet</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="RateWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0" rowspan="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnValid">
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/img/ok_valid.png</normaloff>:/icons/img/ok_valid.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/img/cancel.png</normaloff>:/icons/img/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="searchWidget" name="wdgSearch" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RateWidget</class>
   <extends>QWidget</extends>
   <header>widgets/ratewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>searchWidget</class>
   <extends>QWidget</extends>
   <header>widgets/searchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leNameProject</tabstop>
  <tabstop>leDescription</tabstop>
  <tabstop>btnValid</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnValid</sender>
   <signal>pressed()</signal>
   <receiver>AddProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>released()</signal>
   <receiver>AddProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
